### ISSUE TITLE
Tests fail locally

### ISSUE THREAD

**Issue by jspaezp** [TalusBio/nf-encyclopedia#31]
Hello there,

I noticed that tests fail when running them locally with this error:

```
Command error:
  Unable to find image 'talusbio/nf-encyclopedia:latest' locally
  docker: Error response from daemon: pull access denied for talusbio/nf-encyclopedia, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
```

 because they use this line:
https://github.com/TalusBio/nf-encyclopedia/blob/6b469ea42f3f3b2b2968a5aa39baefa7f72f72a5/conf/test.config#L10

and therefore fail because even though ghcr.io/talusbio/nf-encyclopedia:latest had been pulled, the tag is not matched.

Notably, that line also makes the tests pass in CICD because right before the test runs, the image is built using:

```/usr/bin/docker buildx build --iidfile /tmp/docker-build-push-V0tuRX/iidfile --tag nf-encyclopedia:latest --load --metadata-file /tmp/docker-build-push-V0tuRX/metadata-file .```

https://github.com/TalusBio/nf-encyclopedia/actions/runs/3192943440/jobs/5210985034#step:5:102

So maybe a complete solution would entail looking for env variables and use a file that contains only "nf-encyclopedia" as the container definition, otherwise use "ghcr.io/talusbio/nf-encyclopedia:latest".

LMK what you think!

TLDR:
tests fail if you didnt build the image in the same computer. we can either document it or fix it  :P

**Comment by jspaezp** [comment]
@wfondrie

**Comment by wfondrie** [comment]
I think using environment variables is a great idea! I'm fairly certain that they are substituted without issue in the NextFlow configuration files, so you should just be able to change it to something like:
```
 process.container = "${CONTAINER}" 
```

**Comment by jspaezp** [comment]
I felt like "CONTAINER" was alil 2 generic so I went with `NFE_CONTAINER`. LMK what you think
